projects/storefrontlib/src/cms-components/myaccount/order/replenishment-order-history/replenishment-order-history.component.html

```diff
@@ -0,0 +1,222 @@
+<ng-container *ngIf="replenishmentOrders$ | async as replenishmentOrders">
+  <div class="container">
+    <!-- HEADER -->
+    <div class="cx-replenishment-order-history-header">
+      <h3>
+        {{ 'orderHistory.replenishmentOrderHistory' | cxTranslate }}
+      </h3>
+    </div>
+
+    <!-- BODY -->
+    <div class="cx-replenishment-order-history-body">
+      <ng-container
+        *ngIf="replenishmentOrders.pagination.totalResults > 0; else noOrder"
+      >
+        <!-- Select Form and Pagination Top -->
+        <div class="cx-replenishment-order-history-sort top row">
+          <div
+            class="cx-replenishment-order-history-form-group form-group col-sm-12 col-md-4 col-lg-4"
+          >
+            <cx-sorting
+              [sortOptions]="replenishmentOrders.sorts"
+              [sortLabels]="getSortLabels() | async"
+              (sortListEvent)="changeSortCode($event)"
+              [selectedOption]="replenishmentOrders.pagination.sort"
+              placeholder="{{ 'orderHistory.sortByMostRecent' | cxTranslate }}"
+            ></cx-sorting>
+          </div>
+          <div class="cx-replenishment-order-history-pagination">
+            <cx-pagination
+              [pagination]="replenishmentOrders.pagination"
+              (viewPageEvent)="pageChange($event)"
+              *ngIf="replenishmentOrders.pagination.totalPages > 1"
+            ></cx-pagination>
+          </div>
+        </div>
+        <!-- TABLE -->
+        <table class="table cx-replenishment-order-history-table">
+          <thead class="cx-replenishment-order-history-thead-mobile">
+            <th scope="col">
```

Comment:
these table headers is looking weird in 'tablet' view. Maybe discuss with Jeanne or Bill to make the experience better. I would say maybe a different text or have it do textblabla... if it's too long. 

![Screen Shot 2020-09-23 at 5 19 27 PM](https://user-images.githubusercontent.com/29099466/94070886-fa1d6500-fdc0-11ea-8260-83b39be3ee10.png)

Reply:
UX suggested using a short date format (mm/dd/yy) and trim time from the frequency. need discussion